experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -855,23 +951,43 @@ end
 ###############################################################################
 # more functions
 
+function Base.:+(r::RootSpaceElem, w::WeightLatticeElem)
```

Comment:
I am not really happy with this. There is no clear reason why the return type should be `WeightLatticeElem` and not `RootSystemElem`. Is there a reason this function should exist?
@fingolfin What is your opinion?

In the case we go forward with this, there should be at least a test to verify the return type.

Reply:
I think this is no longer needed. I'll try to remove it.